### DISCUSSION TITLE
A few ideas for temp channels

### DISCUSSION THREAD

**Discussion by computertech1012** [BearWare/TeamTalk5#1511] [General]
Hi,
When you create a temp channel on any server, using QT client, you can configure everything as needed. When the server drops the connection, the client recreate the channel when connection is restored, or if the client lost the connection. I like this feature.
However, I think you should be able to configure a default bitrate to use in the channels that you create, this way you can use the join channel optional feature in order to make your channel, and it will automaticly set the bitrate to stereo instead of mono.
I would also be curious to know, how does the client remember the channel settings and recreate the channel? You can reproduce what I am talking about by creating a channel on the encrypted UK server in the create channel dialog, spesify any bitrate you want, and a password. Then restart your internet and you will notice the channel recreates. I like this a lot especially for the encrypted server.
But I think it's better to also have a default setting that you can set in client, or be able to spesify bitrate within the client in the join channel optional field when connecting to a server.
Let me know if you need more information. Keep up the good work, as always.
Thanks for reading!

**Comment by bear101** [comment]
If your connection to a server is dropped then the Qt client will recreate the channel with its original settings. Also audio codec settings.

Are you saying that bitrate and mono/stereo is not being restored?

**Reply by computertech1012** [reply]
Hi,
It is being restored, I am talking about a setting that will create the channel with the join channel optional fields with stereo or other codex instead of just Mono. Most of the time the QT recreates the channel with all the settings, however it does not restore the operator password.
I am saying that if you use the join channel option in connect to a server dialog there should be a way to spesify bitrate. Also there are still sometimes when a client gives errors and says incorrect channel password at random times during reconnection, but this doesn't happen offen.
I also wanted to make note of another problem. If you are in a server, and you get moved to a password protected channel, moved and not join, then if you lost the connection, the client reconnects but says incorrect channel password. This bug should be fixed as well.
Thanks for reading!
